style.css

```diff
@@ -579,6 +577,8 @@ footer {
   }
 
   #email-contact {
+    display: none;
+    position: absolute;
     width: 100%;
   }
 
```

Comment:
Hello @shella12 😄 I'm going to make a break line for the second one as you suggested, but about the h1 element, I don't see why I need to use another tag for the same phrase, and it's the first heading on the page that's why I chose h1 and separated the two parts of the heading because they needed different colors, so it was not for semantic purposes but for style. Is it ok if I leave the tags as they are?

Reply:
Hi @Movgto,

The second phrase of the heading is basically not the main heading it's a sub heading so for semantic purpose I suggested that you should use another heading tag. And I would highly recommend you to use another heading tag.